CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [0.1.6](https://github.com/acquire-project/acquire-driver-hdcam/compare/v0.1.5...v0.1.6) - 2023-08-10
 
 - Fixes a bug where the output trigger source property was set when it shouldn't be.
+- Fixes a bug where roi bound checking wasn't working.
```

Comment:
Isn't 0.1.6 already released (on August 10th?)? Looks like you'd either need a new release or need to change the date of 0.1.6.

Reply:
going to re-do this release. I'll update the timestamp.